integration/system/version_test.go

```diff
@@ -0,0 +1,28 @@
+package system
+
+import (
+	"os"
+	"testing"
+
+	"github.com/docker/docker/api"
+	"github.com/docker/docker/dockerversion"
+	"github.com/docker/docker/integration-cli/request"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func TestVersion(t *testing.T) {
+	client, err := request.NewClient()
+	require.NoError(t, err)
+
+	version, err := client.ServerVersion(context.Background())
+	require.NoError(t, err)
+
+	assert.Equal(t, api.DefaultVersion, version.APIVersion)
```

Comment:
What if the daemon is being compiled at a different time (or with different version) than the source version being compiled ? Should we check only a `200` response code with valid version format instead ?

Reply:
I don't think we should really expect the test suite to work with a different version of the daemon. 
This is what the old test did so at least it's consistent with what we were testing before.

Reply:
I guess it is a problem with #34862

I can adjust this version check